### PR TITLE
Remove all references to django

### PR DESCRIPTION
**Pull request by jcscottiii**
Replace any functionality with flask

### CODE REVIEW

File: templates/_base.html [hunk 1]
```diff
@@ -74,8 +74,7 @@
 </head>
 
 <body class="loading" data-path="{{current_path}}">
-  {% comment %}
-  {% endcomment %}
+  {# #}
```
Comment (jrobbins) [inline templates/_base.html:77]: This could be deleted.

File: framework/utils.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 from framework import users
 import settings
 
-from django.utils import feedgenerator
+from feedgenerator.django.utils import feedgenerator
```
Comment (jrobbins) [inline framework/utils.py:26]: Thanks for figuring out how to make this still work without needing all of django.  However, the only code that uses this is no longer reachable.  In a follow-up PR we can remove it, and FeatureListXMLHandler and render_atom_feed.

**Review by jrobbins — APPROVED** [review]
Woo-hoo!  Our dependencies are getting lighter.

File: framework/utils.py (hunk 2 above)
Comment (jcscottiii) [reply framework/utils.py:26]: Created https://github.com/GoogleChrome/chromium-dashboard/issues/2366

**Pull request merged**